frontend/src/components/Dashboards/GeneralDashboard/GeneralDashboard.js

```diff
@@ -1,37 +1,41 @@
 import React from 'react';
 import Navbar from "../../Navbar/Navbar";
-import { Tabs } from 'react-bootstrap';
-import { Tab } from 'react-bootstrap';
+import {Tabs} from 'react-bootstrap';
+import {Tab} from 'react-bootstrap';
 import Home from '../../Tabs/Home';
 
 import 'bootstrap/dist/css/bootstrap.min.css';
 import "../DashboardCommonStyles.css";
 import styles from "./GeneralDashboard.module.css";
 
-import { useAuth } from "../../Authentication/FirebaseAuth/FirebaseAuth";
+import {useAuth} from "../../Authentication/FirebaseAuth/FirebaseAuth";
+import NotificationBox from "../../NotificationBox/NotificationBox";
 
 function GeneralDashboard(props) {
 
-    let { currentUser } = useAuth();
-    
+    let {currentUser} = useAuth();
+
     console.log("General dashboard user: ", currentUser);
-    
+
 
     return (
         <div>
-            <Navbar />
+            <Navbar/>
             <div className={styles["general-main-container"]}>
                 <div className={styles["tabs-box"]}>
-                <Tabs className="tabStyle" defaultActiveKey="home">
-                    <Tab eventKey="home" title="Home">
-                        <div className={styles["tab-page"]}>
-                            <Home />
-                        </div>
-                    </Tab>
-                </Tabs>
+                    <Tabs className="tabStyle" defaultActiveKey="home">
+                        <Tab eventKey="home" title="Home">
+                            <div className={styles["tab-page"]}>
+                                <Home/>
+                            </div>
+                        </Tab>
+                    </Tabs>
                 </div>
-            </div>
 
+                <div className={styles["notification-outer-box"]}>
+                    <NotificationBox/>
```

Comment:
A generic message still needs to be shows if no new notifications are available

Reply:
I added it. It's in NotificationBox.js. However I know that if you delete all the notifications that message won't show because I didn't implement that. But if you start with 0 notification, it'll show "No notification"

Reply:
The message should show even after a user has deleted all notifications. You can update the state when notifications get deleted